src/sentry/notifications/notifications/strategies/role_based_recipient_strategy.py

```diff
@@ -52,11 +52,6 @@ def get_role_string(self, member: OrganizationMember) -> str:
         role_string: str = roles.get(member.role).name
         return role_string
 
-    def build_notification_footer_from_settings_url(
-        self, settings_url: str, recipient: User
-    ) -> str:
+    def get_recipient_role_string(self, recipient) -> str:
```

Comment:
No reason why notification footer should be built here, moved it back to the notification class.